clio-client/src/main/scala/org/broadinstitute/clio/client/parser/BaseArgs.scala

```diff
@@ -28,8 +31,8 @@ class BaseParser extends scopt.OptionParser[BaseArgs]("clio-client") {
     .action((token, conf) => conf.copy(bearerToken = token))
     .text("A valid bearer token for authentication.")
 
-  cmd(Commands.addReadGroupBam)
-    .action((_, c) => c.copy(command = Some(Commands.addReadGroupBam)))
+  cmd(Commands.AddReadGroupBam.apply().commandName)
```

Comment:
Here's the weird `apply` usage I was talking about. If our commands were case objects we could just to `AddReadGroupBam.commandName`